ui/src/pages/Feed.svelte

```diff
@@ -276,7 +281,7 @@
                 placeholder="Search Portal"
                 bind:value={positiveFeedPrompt}
                 bind:this={positiveFeedPromptForm}
-                on:keyboardSubmit={handlePromptFeed}
```

Comment:
these aren't textAreas, so they don't have dispatchers. so hotkey submit was broken here. is this the best way to do this?

Reply:
Sure, this would work. If this is the case it might even be worth factoring out the if statement on line 64 into the utils file or something, so that it's not required to change it in a bunch of places if we decide we want the shortcut to be `ctrl + someOtherKey` or remove the shortcut entirely.